video/out/placebo/utils.c

```diff
@@ -52,9 +52,12 @@ static void log_cb_probing(void *priv, enum pl_log_level level, const char *msg)
 
 pl_log mppl_log_create(void *tactx, struct mp_log *log)
 {
+    int log_level = mp_msg_level(log);
+    log_level = log_level == -1 ? PL_LOG_NONE : msg_lev_to_pl_log[log_level];
```

Comment:
nit, maybe `msg_lev_to_pl_log` should be converted to function to encapsulate this logic?